examples/commands-nix/build.sbt

```diff
@@ -1,6 +1,6 @@
 name := "commands-nix"
 
-scalaVersion := "2.12.13"
+scalaVersion := "2.13.5"
 
 libraryDependencies ++= Seq(
   "org.scalacheck" %% "scalacheck" % "1.15.1" % "test",
```

Comment:
this doesn't actually have to do with the changes in this PR, but I happened to notice that the 1.15.1 here (and perhaps in the other examples too?) is out-of-date

Reply:
#766